src/StakeRegistry.sol

```diff
@@ -190,6 +190,16 @@ contract StakeRegistry is StakeRegistryStorage {
         return quorumsToRemove;
     }
 
+    /**
+     * @notice Called by the registry coordinator when all operators for a quorum have been updated
+     * all at once. Updates to the current block timestamp
+     * @param quorumNumber the specific quorum that has been updated
+     */
+    function updateQuorumTimestamp(uint8 quorumNumber) external onlyRegistryCoordinator {
```

Comment:
why not keep this in the reg coor?

Reply:
Agreed

Reply:
I figured the StakeRegistry should be keeping track of the timestamp mapping but we can remove the external call and keep it in the coordinator

Reply:
updating operators means keeping their operator status up-to-date, as well as their listings in each of the registries. for that reason, i think the timestamp is a coordinator concept

Reply:
Moved to reg coordinator https://github.com/Layr-Labs/eigenlayer-middleware/pull/62/commits/6e650e5550e57bd1d071cb42072808d2f5558b70